test/grids.jl

```diff
@@ -40,32 +40,14 @@ function grids_testsuite(AT, FT)
         R = 1
         nedge = 3
 
-        coarse_grid = Raven.cubeshellgrid(R, nedge)
+        coarse_grid = Raven.cubeshellgrid(R)
 
         gm = GridManager(LobattoCell{Tuple{N...},Float64,Array}(), coarse_grid, min_level = 2)
 
         indicator = rand((Raven.AdaptNone, Raven.AdaptRefine), length(gm))
         adapt!(gm, indicator)
 
-        function cubespherewarp(point::SVector{3})
-            p = sortperm(abs.(point))
-            point = point[p]
-
-            ξ = π * point[2] / 4point[3]
-            η = π * point[1] / 4point[3]
-
-            y_x = tan(ξ)
-            z_x = tan(η)
-
-            x = point[3] / hypot(1, y_x, z_x)
-            y = x * y_x
-            z = x * z_x
-
-            point = SVector(z, y, x)[sortperm(p)]
-
-            return point
-        end
-
+        cubespherewarp = Raven.warp(coarse_grid)
```

Comment:
Likewise, I would move this call into `generate` and  apply to coarse grid warping before the passed in warping.